README.md

```diff
@@ -102,4 +102,8 @@ cargo publish -p sqruff
 
 ## Community
 
-Join the sqruff community on [GitHub Discussions](https://github.com/quarylabs/sqruff/discussions) to ask questions, suggest features, or share your projects.
+Join the Quary community on [Slack](https://quarylabs.slack.com/) to ask questions, suggest features, or share your projects. Also feel free to raise any issues in the repository.
```

Comment:
```suggestion
Join the Quary community on [Slack](https://join.slack.com/t/quarylabs/shared_invite/zt-2dlbfnztw-dMLXJVL38NcbhqRuM5gUcw) to ask questions, suggest features, or share your projects. Also feel free to raise any issues in the repository.
```

Reply:
I think this is the correct invite link ^ The other one takes me to a sign in page.